src/openrct2/world/Map.cpp

```diff
@@ -1539,6 +1540,54 @@ void map_update_tiles()
     }
 }
 
+void map_calc_highest_tile_height()
+{
+    gMapHighestTileHeight = 0;
+    for (int32_t i = 0; i < MAX_TILE_TILE_ELEMENT_POINTERS; i++)
+    {
+        TileElement* tile = &gTileElements[i];
+        int32_t base_z = tile->GetBaseZ();
+        if (base_z > gMapHighestTileHeight)
+        {
+            gMapHighestTileHeight = base_z;
+        }
+    }
+}
+
+MapRange map_get_edge_limits()
+{
+    uint8_t rotation = get_current_rotation();
+
+    int32_t x1 = MAP_MINIMUM_X_Y;
+    int32_t y1 = MAP_MINIMUM_X_Y;
+    int32_t x2 = gMapSizeMinus2;
+    int32_t y2 = gMapSizeMinus2;
+
+    switch (rotation)
```

Comment:
We already have lots of rotation functions. Don't add another use the already existing one

Reply:
Is there a function you have in mind that I can use for this? I don't see one that does what I'm doing here with this range.